server/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -631,6 +641,11 @@ def _delete_model_monitoring_stream_resources(
                 error=mlrun.errors.err_to_str(e),
             )
 
+        logger.debug(
+            "Successfully deleted model monitoring stream resources",
+            project_name=project_name,
+        )
```

Comment:
I suggest moving it inside the try clause at (L635) because if there is a failure we don't break the flow, just printing a warning.